### ISSUE TITLE
how to reproduce the Param Softmax with KNN performance

### ISSUE THREAD

**Issue by valencebond** [zhirongw/lemniscate.pytorch#16]
after adding a linear layer of （128，50000）with the same hyperparameters，top1 performance is only 30.
